jetty-ee9/jetty-ee9-maven-plugin/src/main/java/org/eclipse/jetty/ee9/maven/plugin/QuickStartGenerator.java

```diff
@@ -81,15 +80,15 @@ public void setServer(Server server)
         this.server = server;
     }
 
-    public File getWebAppPropsFile()
+    public Path getWebAppPropsFile()
     {
         return webAppPropsFile;
     }
 
     /**
      * @param webAppPropsFile properties file describing the webapp
      */
-    public void setWebAppPropsFile(File webAppPropsFile)
+    public void setWebAppPropsFile(Path webAppPropsFile)
```

Comment:
Rename to `setWebAppPropsPath`.

Reply:
In a different PR focused on the maven plugin.